scripts/setup-local-dev.sh

```diff
@@ -160,7 +160,8 @@ if [ "$os" == "windows" ]; then
     cscript //NoLogo //B //E:jscript ./shortcut.wscript.js
     rm -f ./shortcut.wscript.js
     
-    bash ./makelocal.sh
+    current_dir="$( cd "$( dirname "${BASH_SOURCE[0]}" )" && pwd )"
+    bash "$current_dir/makelocal.sh"
```

Comment:
strange, this already calls `cd `dirname $0`/..` so  https://github.com/c9/core/pull/284/files#diff-8ca9b3b2ddf419a3ab4eb56c8cb71b66R34 so the directory from which setup-local-dev.sh is invoked shouldn't matter. Maybe there's an issue with quoting? Does the directory where you run this have spaces in the name?